cmd/kube-apiserver/app/testing/testserver.go

```diff
@@ -49,9 +50,12 @@ type TearDownFunc func()
 type TestServerInstanceOptions struct {
 	// DisableStorageCleanup Disable the automatic storage cleanup
 	DisableStorageCleanup bool
-
 	// Enable cert-auth for the kube-apiserver
 	EnableCertAuth bool
+	// Skip healthz check and default namespace creation check
+	SkipPostStartChecks bool
```

Comment:
Instead of setting this, can we instead infer this from a non-nil storageversionwrapfunc?  I'd really like to avoid anyone thinking this is a generally safe, non-racy thing to set and no one will want to set a storageversionwrapfunc.